File: app/src/main/java/com/edi_mariyanto/test_api/model/java/UserResponse.java
package com.edi_mariyanto.test_api.model.java;

import com.google.gson.annotations.SerializedName;

public class UserResponse {
    @SerializedName("id")
    private int id;
    private String userName;
    @SerializedName("name")
    private String name;

    public int getId() {

        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/com/edi_mariyanto/test_api/model/UserResponse.kt
package com.example.example

import com.google.gson.annotations.SerializedName // TODO : jika error pada import ini, install library GSON

// hasil generate dari response API https://jsonplaceholder.typicode.com/users
// , kemudian jsonnya di convert menggunakan generator online https://www.json2kt.com/,
// setelah itu menghasilkan 4 kelas dicopy kedalam package / folder model.
data class UserResponse (

  @SerializedName("id"       ) var id       : Int?     = null,
  @SerializedName("name"     ) var name     : String?  = null,
  @SerializedName("username" ) var username : String?  = null,
  @SerializedName("email"    ) var email    : String?  = null,
  @SerializedName("address"  ) var address  : Address? = Address(),
  @SerializedName("phone"    ) var phone    : String?  = null,
  @SerializedName("website"  ) var website  : String?  = null,
  @SerializedName("company"  ) var company  : Company? = Company()

)